CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 cmake_minimum_required(VERSION 3.5.0)
 project(cmake_common_scripts VERSION 0.1.0)
+include(cmake/cmake_tools.cmake)
```

Comment:
Wasn't this part of the original PR and then removed?

Did it turn out this was needed after all?

Reply:
Without it, cmake can't find the configure_package macro used below